python/TestHarness/TestHarness.py

```diff
@@ -1117,7 +1117,6 @@ def parseCLArgs(self, argv):
         hpcgroup = parser.add_argument_group('HPC Options', 'Options controlling HPC execution')
         hpcgroup.add_argument('--hpc', dest='hpc', action='store', choices=['pbs', 'slurm'], help='Launch tests using a HPC scheduler')
         hpcgroup.add_argument('--hpc-host', nargs='+', action='store', dest='hpc_host', metavar='', help='The host(s) to use for submitting HPC jobs')
-        hpcgroup.add_argument('--hpc-pre-source', nargs=1, action="store", dest='hpc_pre_source', metavar='', help='Source specified file before launching HPC tests')
```

Comment:
Does this prohibit other clusters from launching non-apptainer environments?

Reply:
It does for now, yes. I plan on extending this too, but wanted to remove it to have it clean before I went down that path

Reply:
can we do it all in one go? Just this PR I mean?

Reply:
I know we've seen folks using it (like this: https://github.com/idaholab/moose/discussions/25342#discussioncomment-6868719)

Are we talking about the same functionality being lost "for a time"?

Allowing the end-user to source that one file right before the TestHarness actually launches, certainly solves the little nuances we can't anticipate for all clusters. I would like to figure out how to leave this in place, if possible.

Reply:
I just don't want to support it until I can test it